examples/emodel_pipeline_local_python/targets.py

```diff
@@ -6,6 +22,19 @@
     "./ephys_data/C060109A1-SR-C1/" # if using ibw files, please provide the path to the folder containing the .ibw files
 ]
 
+# FEATURE TIMINGS IN FEATURE EXTRACTION
+# ----------------------------------
+# Note regarding feature timings (ton, tmid, tmid2, toff):
+# While these timings are not mandatory parameters, they significantly enhance the reliability of feature extraction.
+# Bluepyefe includes an algorithm for their automatic detection; however, this can occasionally fail, especially with noisy data.
+# Therefore, if known, providing these timings is recommended for increased accuracy and reliability of the results.
+
+# LIQUID JUNCTION POTENTIAL (LJP) CORRECTION
+# ----------------------------------
+# Liquid Junction Potential (ljp) is a correction factor that's applied to the voltage measurements in experimental data.
```

Comment:
looks great! Could you write 'that is', 'It is', 'you are' instead of that's, It's, you're